generated/google_cloud_protobuf/test/encoding_test.dart

```diff
@@ -72,113 +41,6 @@ void main() {
     expect(encodeDouble(double.negativeInfinity), '-Infinity');
   });
 
-  test('enum', () {
```

Comment:
Why are we removing these tests? We should note the removed tests in the commit message.

Reply:
Because I deleted the functions that they are testing. The generator is now doing:

```dart
  field: [for (final i in json['enum']) Enum.FromJson(i)]
```

instead of:

```dart
  field: decodeListEnum(json['enum'], Enum.FromJson)
```